string1.py

```diff
@@ -59,7 +62,12 @@ def both_ends(s):
 
 def fix_start(s):
     # your code here
-    return
+    first_char = s[0]
+    slist = list(s)
+    for i, char in enumerate(slist):
+        if slist[i] == first_char and 1 <= i <= len(s):
+            slist[i] = '*'
```

Comment:
I like this logic! Something to think about: how could you use the .replace() method here instead of a for-loop?